cli/pack/app_dir.go

```diff
@@ -24,7 +24,7 @@ const (
 func initAppDir(appDirPath string, projectCtx *project.ProjectCtx) error {
 	var err error
 
-	log.Infof("Create application dir: %s", appDirPath)
+	log.Infof("Initialize application dir")
```

Comment:
Why did you drop `appDirPath `?

Reply:
Because I think that long tmp directory name in logs is a bad idea. In fact, user doesn't care where application dir is placed in a tmp directory. Path to used tmp directory is logged at the beginning.